testdata/yaml/synthetic-monitoring/check-simple.yaml

```diff
@@ -11,7 +11,7 @@ spec:
     job: grafana-com
     labels: []
     offset: 0
-    probes:
+    probeNames:
```

Comment:
this is a breaking change. A reasonable one, but breaking. Can we make it back-compatible?

Reply:
done